doc/source/HOW-TO/cluster-configuration.rst

```diff
@@ -1,38 +1,72 @@
-******************************
-How to configure Qserv cluster
-******************************
+*******************************************
+How to configure a multi-node Qserv cluster
+*******************************************
 
-Pre-requisite
-=============
+Pre-requisites
+==============
 
-Build/install Qserv on each machine of the cluster, using eups.
+Build/install Qserv and its pre-requisites on all nodes.
 
-Install master and worker nodes
-===============================
+Configure master and worker nodes
+=================================
 
-On each node:
+1. Set up environment on master and all workers:
 
-- setup qserv
-- make fresh run directory
-- edit meta-configuration file
-- fill run directory.
+.. code-block:: bash
 
-Example for master node:
+  source <path to stack>/loadLSST.bash
+  cd <path to respositories>
+  setup -r qserv
+  setup -r qserv_testdata -k
 
-.. code-block:: bash
+2. Configure Qserv on master and all workers:
+
+- Run :code:`qserv-configure.py -i -R <qserv-run directory>`
+
+- Edit :code:`<qserv-run directory>/qserv-meta.conf`
+ 
+  - Change :code:`node_type` to :code:`master` or :code:`worker`
+  - | Change master DNS name to the name of the machine as shown by :code:`hostname -s`.
```

Comment:
change to: "Change master DNS name to the name of the master machine as shown by..."